examples/Db2/README.md

```diff
@@ -0,0 +1,87 @@
+
+# Terraform Module to install and configure DB2 on Openshift
+
+**Module Source**: `github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/Db2`
+
+## Set up access to IBM Cloud
+
+If running these modules from your local terminal, you need to set the credentials to access IBM Cloud.
+
+Go [here](../../CREDENTIALS.md) for details.
+
+### Download required license files
+
+Download required license files from [IBM Internal Software Download](https://w3-03.ibm.com/software/xl/download/ticket.wss) or [IBM Passport Advantage](https://www.ibm.com/software/passportadvantage/) into the  `./files` folder
+```bash
+DB2:
+Part Number : CNB21ML
+Filename    : DB2_AWSE_Restricted_Activation_11.5.zip
+```
+Note: the license key is required only for Advanced DB2 installation
```

Comment:
This sentence should probably be listed under the description on the `inputs` table.  Not here since there isn't a reference to it in this section.  User will be confused what this is referring to.

Reply:
The sentence is removed and added under `db2_standard_license_key`, and I updated the requirement to `No` for a default installation.